Java Migration/Azure_DevOps_Server_2019.md

```diff
@@ -0,0 +1,41 @@
+
+
+Below diagram explains the steps to migrate from Oracle JRE to Azul Zulu OpenJDK for Azure DevOps Server 2019.
+
+![Java Migration flow](https://github.com/msftazdev/Code-Search/blob/msftazdev-patch-1/Java%20Migration/flow1.png)
```

Comment:
Image path hardcoded to your fork, should be relative. Same with the links in the other docs in this PR

Reply:
Fixed them